mido/backends/portmidi.py

```diff
@@ -262,7 +262,7 @@ class Output(PortCommon, BaseOutput):
     def _send(self, message):
         if message.type == 'sysex':
             # Sysex messages are written as a string.
-            string = pm.c_char_p(bytes(message.bin()))
+            string = pm.ctypes.c_char_p(bytes(message.bin()))
```

Comment:
I believe it should be `ctypes.c_char_p` instead of `pm.ctypes.c_char_p` since `ctypes` is available in the context.

Reply:
change applied.